sympy/calculus/util.py

```diff
@@ -60,20 +60,20 @@ def continuous_domain(f, symbol, domain):
     try:
         sings = S.EmptySet
         if f.has(Abs):
-            sings = solveset(1/f, symbol, domain)
+            sings = solveset(1 / f, symbol, domain)
```

Comment:
we leave spaces around `+` and `-` but not `/` and `*`. Can you remove this changes?